torchaudio/functional/functional.py

```diff
@@ -2398,14 +2398,19 @@ def convolve(x: torch.Tensor, y: torch.Tensor, mode: str = "full") -> torch.Tens
     .. _convolution:
         https://en.wikipedia.org/wiki/Convolution
     """
-    _check_shape_compatible(x, y, allow_broadcast=False)
+    _check_shape_compatible(x, y, allow_broadcast=True)
```

Comment:
After this change, `_check_shape_compatible` is always called with `allow_broadcast=True`, so can we remove the arg from `_check_shape_compatible`?